src/plugins/pdfPlayer/style.scss

```
@@ -4,7 +4,7 @@
     width: 100%;
     overflow: none;
     z-index: 100;
-    background: #fff;
+    background: #000000;
```

```suggestion
    background: #000;
```

Our stylelint rules prefer the short color format when possible.

---

Okay got it! I've updated the PR